Macropad_MIDI_Tester/code.py

```diff
@@ -0,0 +1,93 @@
+# SPDX-FileCopyrightText: 2021 John Park for Adafruit Industries
+# SPDX-License-Identifier: MIT
+# Macropad MIDI Tester
+# Play MIDI notes with keys
+# Click encoder to switch modes
+# Turn encoder to adjust CC, ProgramChange, or PitchBend
+from adafruit_macropad import MacroPad
+from rainbow import colorwheel
```

Comment:
This has been updated to `rainbowio`. Please update it here. Thanks!

Reply:
on it